protocol-dashboard/src/services/Audius/sdk.ts

```diff
@@ -1,11 +1,64 @@
-import { sdk } from '@audius/sdk'
+import {
+  DiscoveryNodeSelector,
+  EntityManager,
+  developmentConfig,
+  productionConfig,
+  sdk,
+  stagingConfig
+} from '@audius/sdk'
 
-let audiusSdk = null
+const SDK_LOADED_EVENT_NAME = 'AUDIUS_SDK_LOADED'
+let instance = null
+let isReady = false
 
-const initAudiusSdk = () => {
-  audiusSdk = sdk({ appName: 'Audius Protocol Dashboard' })
+const initAudiusSdk = ({
+  signTransaction,
+  getAddress
+}: {
+  signTransaction: (data: any) => Promise<string>
+  getAddress: () => Promise<string> | string
+}) => {
+  const env = import.meta.env.VITE_ENVIRONMENT
+  const bootstrapConfig =
+    env === 'development'
+      ? developmentConfig
+      : env === 'staging'
+      ? stagingConfig
+      : productionConfig
+  const {
+    discoveryNodes,
+    entityManagerContractAddress,
+    web3ProviderUrl,
+    identityServiceUrl
+  } = bootstrapConfig
+  const dnSelector = new DiscoveryNodeSelector({
+    bootstrapServices: discoveryNodes,
+    initialSelectedNode: 'https://discoveryprovider.staging.audius.co'
```

Comment:
Do we need to pass the initialSelectedNode? Since we are passing bootstrapServices it should just pick one from there initially right

Reply:
Oh good catch. I hard coded this for testing